Directory.Build.props

```diff
@@ -6,6 +6,7 @@
       tell Microsoft.Common.props not to import Directory.Build.props again
     -->
     <ImportDirectoryBuildProps>false</ImportDirectoryBuildProps>
+    <TargetFrameworkVNext>netcoreapp5.0</TargetFrameworkVNext>
```

Comment:
Should this rather be `DefaultTargetFramework` or `CurrentTargetFramework`?

VNext will be very confusing in release/servicing branches or in the source build.

Reply:
Agreed.  Perhaps `$(netcoreappCurrent)` so that it parses well when seen in `<TargetFrameworks>`